common/src/main/java/org/apache/celeborn/common/meta/FileInfo.java

```diff
@@ -110,6 +110,11 @@ public synchronized long getFileLength() {
     return chunkOffsets.get(chunkOffsets.size() - 1);
   }
 
+  // for mappartition, a file only has one chunk.
+  public synchronized void updateFileLength(long fileLength) {
```

Comment:
I think we'd better to set chunk size to infinity for map partition to avoid writing chunk offsets, instead of introducing this branch logic.

Reply:
At before, fileLength equals to `chunkOffsets.get(chunkOffsets.size() - 1)` (which is updated when bytesFlushed are increased by  `8M`(by defaults) ), but now it equals to` bytesFlushed `( which is updated when bytesFlushed are increased by `256k`(by defaults) ).